docs/edge-compute/edge-agent-installation.md

```diff
@@ -38,15 +38,25 @@ Verify that you now have the key with the fingerprint `9DC8 5822 9FC7 DD38 854A
 sudo apt-key fingerprint 0EBFCD88
 ```
 
-Use the following command to set up the stable repository:
+Use the following command to set up the stable repository (`amd`):
 
 ```console
+# Amd
 sudo add-apt-repository \
    "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
    $(lsb_release -cs) \
    stable"
 ```
 
+If you came here while doing [the walkthrough](/edge-compute/walkthrough/), you'll want the `arm` stable repository:
```

Comment:
"If you came here while doing the walkthrough, or if you are using an ARM device, " might be more clear